cmd/tctest/cli/cmds.go

```diff
@@ -292,6 +297,7 @@ Complete documentation is available at https://github.com/katbyte/tctest`,
 		"queue-timeout": "",
 		"run-timeout":   "",
 		"latest":        "TCTEST_LATESTBUILD",
+		"skip-queue":  "",
```

Comment:
could we add Ann environment var for this? as I'm going to set it to always be true 😬 
```suggestion
		"skip-queue":  "TCTEST_SKIP_QUEUE",
```

Reply:
Good idea, done!